cogs5e/models/homebrew/bestiary.py

```diff
@@ -334,7 +334,7 @@ def _monster_factory(data, bestiary_name):
                                              resist=data['stats']['damageResistances'],
                                              immune=data['stats']['damageImmunities']))
 
-    return Monster(name=data['name'], size=data['stats']['size'], race=data['stats']['race'],
+    return Monster(name=data['name'], size=data['stats']['size'], creature_type=data['stats']['race'],
```

Comment:
The argument name in the `gamedata.Monster` constructor wasn't changed, so this will cause ~~an error~~ the data to be lost (it silently drops extra kwargs)